src/pymovements/utils/parsing.py

```diff
@@ -371,6 +378,7 @@ def parse_eyelink(
     pre_processed_metadata['data_loss_ratio'] = data_loss_ratio
     pre_processed_metadata['data_loss_ratio_blinks'] = data_loss_ratio_blinks
     pre_processed_metadata['total_recording_duration_ms'] = total_recording_duration
+    pre_processed_metadata['tracked_eyes'] = tracked_eyes
```

Comment:
The dictionary field should then also be renamed to `tracking_configs`.

Also make sure that the first match from `EYELINK_META_REGEXES` is included too. Otherwise the entry will be empty if there were no changes in tracking configuration.